ct.yaml

```diff
@@ -4,12 +4,10 @@
 remote: origin
 target-branch: main
 chart-dirs:
-  - chart
-charts: chart
```

Comment:
Specifying `charts` disables version increment test and hence it needs to be removed.